@here/harp-mapview/lib/Tile.ts

```diff
@@ -1285,6 +1285,34 @@ export class Tile implements CachedResource {
                     materials[techniqueIndex] = material;
                 }
 
+                // Modify the standard textured shader to support height-based coloring.
+                if (
+                    isStandardTexturedTechnique(technique) &&
+                    technique.displacementMap !== undefined
```

Comment:
What if the displacementMap is set in the stylesheet?
You should either add a bool "useHeightColoring" or maybe it's time to introduce a dedicated TerrainTechnique.

Reply:
This was a overlook on my behalf, I forgot to add ``&& technique.heightBasedColor === true``. Should be fixed now.